mixer/template/sample/template.gen_test.go

```diff
@@ -969,12 +999,24 @@ func TestProcessReport(t *testing.T) {
 							Value:          "1",
 							Dimensions:     map[string]string{"s": "2"},
 							Int64Primitive: "54362",
+							DnsName:        `"myDNS"`,
+							Duration:       "request.duration",
+							EmailAddr:      `"myEMAIL"`,
+							IpAddr:         `ip("0.0.0.0")`,
+							TimeStamp:      "request.timestamp",
+							Uri:            `"myURI"`,
```

Comment:
Similarly, this should be a valid URL.

Reply:
Will leave this as is since we are not using the url.URL. And as soon as we use url.URL, this test should fail.